app/Main.hs

```diff
@@ -172,11 +202,12 @@ data Mode
     Check
   deriving (Eq, Show)
 
--- | Configuration for how to account for default-extension
--- from .cabal files
-data CabalDefaultExtensionsOpts = CabalDefaultExtensionsOpts
+-- | Configuration for how to use .cabal files
```

Comment:
"to use" is a bit too vague. Let's say "what kind of information to extract from .cabal files".

Reply:
Fixed!